docker-compose.yml

```diff
@@ -1,6 +1,8 @@
 services:
   app:
     build: .
+    env_file:
```

Comment:
This doesn't need to be explicitly defined, .env is the default.

Reply:
Oops, I was wrong here. If this gets removed, then the variables have to at least be explicitly declared in the environment section which I prefer anyways for the sake of being explicit in the docker-compose configuration.